ios-base/Makefile

```diff
@@ -0,0 +1,5 @@
+init:
```

Comment:
I would like you to write a getting started for iOS section in README.md.

In addition, is this file for GNU make? If so, please note it as a required dependency.

Reply:
> I would like you to write a getting started for iOS section in README.md.

In this case, README.md represents Project's README.md ? not ios-bases/README.md?

Reply:
I think it seems better to use `ios-base/README.md`. 
Because Android Engineers will do `make init` without thinking 😄 
But it is good to link `ios-base/README.md` from `root/README.md`

Reply:
I will write `README.md` at `ios-base/README.md` .
I think it is natural. 🤔

Reply:
Agreed. It doesn't sound that we need to put the section I've mentioned into `/path/to/repo/README.md`. `/path/to/repo/ios-base/README.md` is fine.

Reply:
I added README.md and update Project's README.md! Please check it. 🙏 
https://github.com/DroidKaigi/conference-app-2020/pull/447/commits/c465d8c6b23f23c75e9317236ddf56f9dfe73ce0